### DISCUSSION TITLE
Does having this nullify the need for having Animatica, if one wants to use it's features?

### DISCUSSION THREAD

**Discussion by ChurlOat** [MoreMcmeta/core#70] [Core Dev Questions]
As title says.

**Comment by soir20** [comment]
Both Animatica and MoreMcmeta support animated textures using OptiFine `.properties` files. My understanding is that Animatica doesn't add any custom features, so MoreMcmeta should support everything that Animatica does. If something isn't supported in MoreMcmeta, I would consider that a bug.

I wouldn't recommend using both Animatica and MoreMcmeta because both mods will try to animate the textures. I haven't personally tested them together, but my guess is that you'd run into issues.
